src/CoreWCF.ConfigurationManager/tests/CoreWCF.ConfigurationManager.Tests.csproj

```diff
@@ -39,6 +39,11 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="7.0.5" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
```

Comment:
Can we do something smarter using wildcards and string slicing so we don't need to add a new conditional ItemGroup each time? Could we use string substrings on the target framework to extract out the version number of the tfm and then specify the version as `$(DotNetVersion).*` to pull in the latest version?

Reply:
@mconnew I did not succeeded  yet to "template" this or implement a kind of for loop in ItemGroup based of a PropertyGroup value.

Reply:
done!